calc_flow2 and jupyter report feature

dz24 opened the pull request:
Adds 

* calc_flow2:  it creates a dictionary based on individual replica, necessary for `calc_flow_op`. I am also not too sure, but I get different results to `inft calc_flow` (but I was too lazy to look into that code and just wrote a new one from scratch...)  
* calc_flow_op: Follows one replica across ensembles, and plot its order.txt
* sim_acc: simple function that plots the acceptance probably of MC moves in individual ensembles.
* Add `async_postprocess.py` script, in case `inft trjcat` have to be done for e.g. 10000 paths, this script define workers to do e.g. 20 of these simultaneously.
* Fixed bug introduced in https://github.com/infretis/inftools/pull/33, namely `inft wham` fails for `*toml` files generated by `inft combine_data` that does not contain  `tis_set` in `config["simulation"]["tis_set"]`.
* Add jupyter report feature, where a template is copied into the current dir by the use of `inft report`.

lukasbaldauf commented:
Would it be possible to add the points above to the function docstring? There is no description when using `inft -h` or `inft calc_flow2 -h`

dz24 commented:
You're right, I forgot about that. I will update them when i have time.

mbocus commented:
As discussed, I have prepared a complete script to compute free energy surfaces which is based on `ufree_calc` but should be more flexible.
https://github.com/infretis/inftools/pull/30

dz24 commented:
Could you check out `inft report`? @lukasbaldauf 

It is very rough but it should work I think. Then we can merge it in make it better through new PRs.

dz24 commented:
This latest git commit tries to fix the failing github unittest pipeline by adding `infretis` as a dependency.

`test/test_initial_paths/test_initial_path_from_iretis.py` failed locally too, due to the nonexistent trajectories `run0/0/accepted/0` etc. I added them, and changed the `infretis0.toml` interface values a bit to add the same trajectories (the default ones added a different traj to the last ensemble for some reason)

added the max_op visualization to the report

lukasbaldauf commented:
> `test/test_initial_paths/test_initial_path_from_iretis.py` failed locally too, due to the nonexistent trajectories `run0/0/accepted/0` etc. I added them, and changed the `infretis0.toml` interface values a bit to add the same trajectories (the default ones added a different traj to the last ensemble for some reason)

Thanks, I previously added a check that assures the trajectory files from traj.txt actually exist in the accepted/ folder, so the way you added them here is what was needed. I also looked at why the wrong trajectories were added, and the fix here is also correct.

For the report, I had to modify report.py to be able to run it:

```diff
-from . import report
+from . import __package__ as report_pkg
 JUP_REPORT = "infretis_report.ipynb"
-INP_FILE = impresources.files(report) / JUP_REPORT
+INP_FILE = impresources.files("inftools.report") / JUP_REPORT
```
* I see the calc_simtime function is missing?
* I tried to use the report on data from infinit, with combo.toml and combo.txt, which caused some keywords to be missing, but I see you fixed this

lukasbaldauf commented:
I changed a small part in `report.py`, I think this is preferable to what I initially suggested.

I have one last question: how is `calc_flow2` different from `calc_flow`?
Maybe a paragraph after the main docstring in `calc_flow2` can say something about that. If one runs `inft calc_flow2 -h`, one gets this additional information, making the tool easier to use and understand. Maybe also for `calc_flow_op`.

But other than that I think it looks very good, I think this jupyter notebook report feature is interesting.
